Jenkinsfile

```diff
@@ -22,13 +22,13 @@ pipeline {
       steps {
           sshagent ( ['projects-storage.eclipse.org-bot-ssh']) {
           	script {
-          		def VERSION='master'
-          		if (changeRequest()) {
-          			VERSION=BRANCH_NAME
-          		}
+				def VERSION = BRANCH_NAME
+				if (VERSION.matches("v\\d\\.\\d\\.x")) {
+					VERSION = VERSION.substring(1)
```

Comment:
I think we can deploy artefact of PR in a separated folder. So rather than using the nighlty folder, we can use a new folder (PR for example). This will ease deleting PR artefacts.

What do you think about this ?

Reply:
It won't be easier since new branches without PR are automaticaly built.
Moreover I already set up a cleanning job.